src/server/api/routers/study.ts

```diff
@@ -192,11 +222,11 @@ export const studyRouter = createTRPCRouter({
         .mutation(async ({ ctx, input }) => {
             const study = await prisma.study.findUnique({ where: input });
 
-            if ( ! study ) {
```

Comment:
I've seen some formatting changes, i think there should be like some specific rules set in the project for formatting so we won't update all the lines if someone has different formatting settings